lib/command_cursor.js

```diff
@@ -90,7 +90,7 @@ class CommandCursor extends Cursor {
   /**
    * Set the batch size for the cursor.
    * @method
-   * @param {number} value The batchSize for the cursor.
+   * @param {number} value The number of documents to return per batch. See {@link https://docs.mongodb.com/manual/reference/command/find/|find documentation}.
```

Comment:
I think we should be more explicit here, maybe calling it the "find command documentation"